res/xml/preferences_video.xml

```diff
@@ -93,7 +93,7 @@
             android:title="@string/preferences_display_resume_box" />
 
     </PreferenceCategory>
-    <PreferenceCategory android:title="@string/preferences_user_interface" android:key="category_leanback_user_interface" >
+    <PreferenceCategory android:title="@string/preferences_leanback_user_interface" android:key="category_leanback_user_interface" >
```

Comment:
I will not merge this change since the setting applies both to AndroidTV and phones/tablets. This is thus not intended to TV only.